src/App.tsx

```diff
@@ -1,23 +1,21 @@
+
 import React from 'react'
 
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom'
 
 import './index.css'
-import Login from './components/login/Login'
 import MainPage from './components/mainPage/MainPage'
+import Login from './components/login/Login'
 
 function App() {
-
-
-
   return (
-    <Router >
+    <Router>
       <Routes>
-        <Route path='/' element={<Login />} />
-        <Route path='/home' element={<MainPage /> } />
+        <Route path='/login' element={<Login />} />
+        <Route path='*' element={<Login/>} />
+        <Route path='/home' element={<MainPage />} />
```

Comment:
Are you going to take care of private routes  at this branch or you will do it later?

Reply:
I will do it letter.